app/js/utils/github.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var remote            = window.require('remote');
+var pjson             = remote.require('./package.json');
+var rp                = window.require('request-promise')
+var _                 = require('lodash')
+
+var GitHub = function(options) {
+  this._repo       = 'http://github.com/gillesdemey/Cumulus'
+  this._endpoint   = 'https://api.github.com/repos/gillesdemey/cumulus'
+}
+
+GitHub.prototype.getRepoUrl = function() {
+  return this._repo;
+}
+
+GitHub.prototype.checkForUpdates = function() {
+  return this.getLatestRelease().then(function(response) {
+    var tag = response.tag_name.replace(/v/g, '')
+
+    if(pjson.version == tag) {
+      return Promise.resolve('Cumulus is currently up-to-date with the most recent version.')
+    } else {
+      return Promise.reject('Cumulus is currently not up-to-date.')
```

Comment:
I would change this to a more informative message like
```
A new version of Cumulus is available.
Go to http://gillesdemey.github.io/Cumulus/ to download the latest version.
```

And hopefully we can create a clickable link out of the url. :)